src/Resources/views/Block/block_admin_preview.html.twig

```diff
@@ -0,0 +1,85 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+{% extends sonata_block.templates.block_base %}
+
+{% set translation_domain = settings.translation_domain ?? admin.translationDomain %}
+
+{% block block %}
```

Comment:
Did we name the block „block“ ? Really? 😱

Reply:
Yes, I'm affraid: https://github.com/sonata-project/SonataAdminBundle/blob/af8723a2b657e4bff1a902edefa251304e3b558c/src/Resources/views/Block/block_stats.html.twig#L16